components/SummaryContainer.tsx

```diff
@@ -45,7 +45,8 @@ export default function SummaryContainer(props: {
 
         <div className={`${styles.summaryComponent} ${styles.sharedCompProps}`}>
           <div className={styles.summaryCompHeader}>
-            <h4 className={styles.summaryStyle}>Repos Overview</h4>
+
+          <h3>{`Total Repos (${props.rankArray.green + props.rankArray.yellow + props.rankArray.red})`}</h3>
```

Comment:
Think this needs to have the same header style as the 'Overall' on the left; I think all three headers for the summary components have different font-sizes, would look a bit better if these were consistent. I know all of us contributed to this but just thought it was something to note to fix later